__tests__/app.test.js

```diff
@@ -321,4 +322,82 @@ describe("NC-Games app", () => {
         });
     });
   });
+  describe("POST /api/reviews/:review_id/comments", () => {
+    test("Responds with status 201 and the new comment with all the right properties.", () => {
+      const newComment = {
+        username: "philippaclaire9",
+        body: "Even aliens like this game!",
+      };
+      return request(app)
+        .post("/api/reviews/3/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comment).toEqual({
+            comment_id: 7,
+            body: "Even aliens like this game!",
+            votes: 0,
+            author: "philippaclaire9",
+            review_id: 3,
+            created_at: expect.any(String),
+          });
+        });
+    });
+    test.only("Responds with status 400 Bad Request error, when passed the wrong username.", () => {
+      const newComment = {
+        username: "SOMEBODY",
```

Comment:
wrong username would be a 404 as the user doesn't exist. Bad request would be for an incorrect datatype or if the username is missed completely